open-company/open-company-web
# Company profile

card: https://trello.com/c/VOGf3ehF

test:
- create a new company via REPL or cURL
- go to [/NEW-COMPANY-SYMBOL](http://localhost:3449/NEW-COMPANY-SYMBOL)
- edit company data (make sure you test the last field where you can add custom links)
- refresh the page and check the data are still there
- tap _Reports_ on the left sidebar
- tap the plus button to add a new report
- create a new report
- edit the new report
- refresh the page
- the data were saved? good!
